packages/eslint-plugin/src/rules/no-empty-lifecycle-method.ts

```diff
@@ -67,12 +63,6 @@ export default createESLintRule<Options, MessageIds>({
                     interfaceName,
                     fixer,
                   ),
-                  getImportRemoveFix(
```

Comment:
I'm proposing to remove it because we can't guarantee that the import is **only** being used for this class, in other words it might be being used in another(s) class(es) also in the same file and so we could probably be suggesting a code that does not compile. Does that make sense for you, @JamesHenry?

Reply:
Can we do a simple string-based check on the file contents for that?

If there is more than one occurrence of the string, we leave it (yes, this _could_ have false positives, e.g. from comments, but still better than never removing it for the common case), if there is only one occurrence (the import) then we remove it?

WDYT?

Reply:
Indeed! I've submitted this check, PTAL!